org.metaborg.spoofax.meta.core/src/main/java/org/metaborg/spoofax/meta/core/pluto/util/StrategoExecutor.java

```diff
@@ -137,7 +137,9 @@ public ExecutionResult executeCLI(Arguments arguments) {
             }
             return new ExecutionResult(false, tracker.stdout(), tracker.stderr());
         } finally {
-            dr_scope_all_end_0_0.instance.invoke(context, context.getFactory().makeTuple());
+            SSLLibrary op = (SSLLibrary) context.getOperatorRegistry(SSLLibrary.REGISTRY_NAME);
```

Comment:
You should skip clearing those tables when, for whatever reason, there is no `SSLLibrary` in the context (right now it would throw a `ClassCastException`). Something like: 

```
final @Nullable OperatorRegistry op = context.getOperatorRegistry(SSLLibrary.REGISTRY_NAME);
if(op != null) {
    final SSLLibrary sslLibrary = (SSLLibrary) op;
    sslLibrary.getDynamicRuleTable().clear();
    sslLibrary.getTableTable().clear();
}
```